src/git-auth-helper.ts

```diff
@@ -126,10 +127,10 @@ class GitAuthHelper {
 
   async configureGlobalAuth(): Promise<void> {
     // 'configureTempGlobalConfig' noops if already set, just returns the path
-    const newGitConfigPath = await this.configureTempGlobalConfig()
+    await this.configureTempGlobalConfig()
```

Comment:
The path to the global config was previously needed to replace the placeholder-token. Now the token is written to a separate file, not the global config file directly.

Note, the placeholder-token is related to avoiding the actual token on the command line, when calling the git CLI.